docs/_sidebar.md

```diff
@@ -5,6 +5,7 @@
     - [Local development setup](local-dev-setup.md)
     - [Regular developer tasks](dev-maintenance.md)
     - [Contributing a module](contribute-module.md)
+    - [Update an older module ](migrate-module.md)
```

Comment:
I think the title "Migrate an existing module" would be more accurate here.

Reply:
IBM is specific about the term migrate. We could use `port`
> **Migrate:** To move a product to a different platform, environment, or operating system.
    _Usage_ This is not an approved use of this term. Use "port."